ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts

```diff
@@ -621,4 +621,91 @@ test.describe('Attachment Point Tool', () => {
       await selectTopPanelButton(TopPanelButton.Undo, page);
     }
   });
+
+  test('Verify that the attachment point visualization matches the rendering to PNG/SVG mockup(on several structures)', async ({
+    page,
+  }) => {
+    /*
+    Test case: EPMLSOPKET-15516, EPMLSOPKET-15517, EPMLSOPKET-15522
+    Description: Openeded structures are displayed with the correct attachment points.
+    */
+    await openFileAndAddToCanvas(
+      'Molfiles-V2000/three-examples-attachment-points.mol',
+      page,
+    );
+  });
+
+  test('Verify that the attachment point visualization matches the rendering to PNG/SVG mockup(on one structure)', async ({
+    page,
+  }) => {
+    /*
+    Test case: EPMLSOPKET-15516, EPMLSOPKET-15518
+    Description: Openeded structure are displayed with the correct attachment points.
+    */
+    await openFileAndAddToCanvas(
+      'Molfiles-V2000/four-attachment-point.mol',
+      page,
+    );
+  });
+
+  test('Verify that changing primary attachment points to secondary updates the visualization', async ({
+    page,
+  }) => {
+    /*
+    Test case: EPMLSOPKET-15519
+    Description: Visualization updates accordingly, displaying 
+    the attachment point labels near the curve line.
+    */
+    await openFileAndAddToCanvas('simple-chain.ket', page);
+    await selectNestedTool(page, RgroupTool.ATTACHMENT_POINTS);
```

Comment:
My proposal is to add a function for setting attachment point for atom. It can be something like (fast solution)
```
async function setAttachmentPoints(
  page: Page,
  atom: { label: Atom, index: number },
  primary: boolean,
  secondary?: boolean
) {
 await clickOnAtom(page, atom.label, atom.index);
 const primaryAttachmentPoint = await page.getByLabel(AttachmentPoint.PRIMARY);
 const secondaryAttachmentPoint = await page.getByLabel(AttachmentPoint.SECONDARY);

  if (primary) {
     primaryAttachmentPoint.check();
  } else {
     primaryAttachmentPoint.uncheck();
  }

  if (secondary) {
     secondaryAttachmentPoint.check();
  } else {
     secondaryAttachmentPoint.uncheck();
  }

  await pressButton(page, 'Apply');
}
```